pkg/services/postgresql/all_in_one_types.go

```diff
@@ -1,25 +1,21 @@
-package postgresqldb
+package postgresql
 
 import "github.com/Azure/open-service-broker-azure/pkg/service"
 
 // AllInOneProvisioningParameters encapsulates non-sensitive dbms AND database
 // PostgreSQL-specific provisioning options
 type AllInOneProvisioningParameters struct {
-	ServerProvisioningParameters
-	DatabaseProvisioningParameters
+	DBMSProvisioningParameters     `json:",squash"`
```

Comment:
The `mapStructure` library that we use gets confused if this annotation isn't present wherever we use composition like this.